lib/Ohmbrewer_Temperature_Sensor.h

```diff
@@ -178,6 +184,11 @@ namespace Ohmbrewer {
              */
             Probe* _probe;
 
+            /**
```

Comment:
Sure, thats fine, but I don't see how we are writing to EEPROM any differently. I am only reading here, and that too from the boolean, so there should be no real difference between the method you are suggesting other than it being "cleaner"